scheduler/docker/volume-plugin.md

```diff
@@ -54,12 +54,31 @@ Options:
     --opt io_priority=value             IO Priority: [high|medium|low] (default: "low")
     --opt sticky                        sticky volumes cannot be deleted until the flag is disabled [on | off]
     --opt snap_interval=value           snapshot interval in minutes, 0 disables snaps (default: 0)
+    --opt snap_schedule=value           snapshot schedule specification. PX 1.3 and higher. (See "Scheduled snapshots" below)
     --opt aggregation_level=value       aggregation level: [1..3 or auto] (default: "1")
     --opt nodes=value                   comma-separated Node Id(s)
 
 ```
 #### Snapshot
 
+**Scheduled snapshots**
+
+Scheduled snapshots are only available in Portworx 1.3 and higher.
+
+Use the _snap_schedule_ option to specify the snapshot schedule.
+
+{% include snap-schedule-format.md %}
+
+Some examples of snapshots schedules are:
+
+* snap_schedule="periodic=60,10"
```

Comment:
for inline spec i think "," is to be replaced by "#".
and you can specify multiple schedules together:

snap_schedule="daily=12:00#4;monthly=15@12:00"